src/renderer/components/dock/create-resource.tsx

```diff
@@ -21,6 +25,39 @@ interface Props {
 @observer
 export class CreateResource extends React.Component<Props> {
   @observable error = "";
+  templates:GroupSelectOption[] = [];
+
+  componentDidMount(){
+    let rootPaths: any[]= [];
+
+    for (const store of this.templatesStores) {
+      rootPaths = rootPaths.concat(fs.readdirSync(store).map(value => {return {root: store, value};}));
+    }
+    const ungroupedFiles: any[] = [];
+
+    rootPaths.forEach((rootPath, index) => {
+      const absRootPath = path.join(rootPath.root, rootPath.value);
+
+      const isOverrided :boolean = rootPaths.map(v => v.value).indexOf(rootPath.value) < index;
```

Comment:
I don't think we should override the defaults. How about just adding them to the end?

Reply:
How about adding a badge to templates added by the user?
![image](https://user-images.githubusercontent.com/53330707/111327183-97799780-8686-11eb-816e-74dd88825277.png)
It seems I should work on alignment, it is just a prototype

Reply:
Could you have both then? `ConfigMap user` and `ConfigMap Lens`?

Reply:
So built-in templates will be not marked, only created by the user, also the group can include two templates with the same name but are from different sources (lens/user), no overrides at all, right?

Reply:
Oh I meant the built in ones to be marked but not marked is better, I agree.

If you want to be the user provided ones with a badge instead of at the end, sure and sort them might be good idea.

Reply:
yes, I think we should highlight only user templates. There is no sense to give a badge to all items because it will make identifying them more difficult. 
About sorting: 
I'm going to place groups on top which was created by the user (and not exist in built-in group list)
In other cases just sorting by the label, what do you think about it?

Reply:
That sounds good.